modules/nft/src/lib.rs

```diff
@@ -308,8 +313,11 @@ decl_module! {
 			<T as Trait>::Currency::unreserve(&owner, data.deposit);
 			<T as Trait>::Currency::transfer(&owner, &dest, data.deposit)?;
 
-			// Skip two steps until pallet_proxy is accessable
-			// pallet_proxy::Module<T>::remove_proxies(owner)
+			// remove all proxies of class owner
+			// `remove_proxies` of pallet_proxy is still private
+			let (_, proxy_deposit) = pallet_proxy::Proxies::<T>::take(&class_info.owner);
+			<T as pallet_proxy::Trait>::Currency::unreserve(&class_info.owner, proxy_deposit);
```

Comment:
Actually maybe we don't need to remove proxy here. it can be done by making another extrinsic on proxy pallet directly

Reply:
Followup https://github.com/AcalaNetwork/Acala/issues/549